src/index.ts

```diff
@@ -1,4 +1,5 @@
-export { Runtime } from './runtime/Runtime';
+export { Runtime , RuntimeType} from './runtime/Runtime';
+export { WechatMiniGameRuntime } from './runtime/WechatMiniGameRuntime';
```

Comment:
Without WechatMiniGameRuntime there's no way to invoke getGameCanvas in client code.